xrpl4j-core/src/main/java/org/xrpl/xrpl4j/model/client/accounts/AccountObjectsRequestParams.java

```diff
@@ -61,6 +61,7 @@ static ImmutableAccountObjectsRequestParams.Builder builder() {
   static AccountObjectsRequestParams of(Address classicAddress) {
     return builder()
       .account(classicAddress)
+      .ledgerSpecifier(LedgerSpecifier.CURRENT)
```

Comment:
Why is `CURRENT` the chosen default here?

Reply:
This was actually a bug that I found that we didn't have test coverage for. Before this change, `AccountObjectsRequestParams.of` would throw an `IllegalStateException` always. Many of the other request params  classes (ie `AccountInfoRequestParams`) have a similar method, and they all default to `LedgerSpecifier.CURRENT`, so this is consistent with those.

Reply:
That makes sense -- however, shouldn't we choose `VALIDATED` in the default case?

Reply:
After speaking with @sappenin, we decided to potentially change the default from `CURRENT` to `VALIDATED` in v4 (see #426), as changing this default would require changing other defaults, which would be a breaking change.